src/services/search.js

```diff
@@ -4,56 +4,61 @@ import request from '../utils/request';
 export async function queryIndexMapping(params) {
   const { datastoreConfig, indices } = params;
 
-  const endpoint = `${datastoreConfig.elasticsearch}/${datastoreConfig.prefix}${
-    datastoreConfig.run_index
-  }${indices[0]}/_mappings`;
+  const url = `${datastoreConfig.prefix}${datastoreConfig.run_index}${indices[0]}/_mappings`;
```

Comment:
This isn't a `url`. It's not really an "index" either since you're pulling the template mappings for one index.

I'm not quite sure _what_ to call this, actually; although in terms of the interim pass-through Elasticsearch API, we call it "indices".

Reply:
Perhaps we should separate this out a bit:

 * The code forms a full index name: `${datastoreConfig.prefix}${datastoreConfig.run_index}${indices[0]}`
 * The `url` is really trying to specify the `_mapping` API for a given index